docs-2.0/3.ngql-guide/7.general-query-statements/6.show/6.show-hosts.md

```diff
@@ -8,7 +8,12 @@ The `SHOW HOSTS` statement shows the host and version information of Graph Servi
 SHOW HOSTS [GRAPH | STORAGE | META];
 ```
 
-If you return `SHOW HOSTS` without the service name, it will show the host information of Storage Service, as well as the leader number, leader distribution, and partition distribution.
+!!! note
+
+    - If you return `SHOW HOSTS` without the service name, it will show the host information of Storage Service, as well as the leader number, leader distribution, and partition distribution.
+  
+    - For a Nebula Graph cluster installed with the source code method, the version of the cluster will not be displayed in the output after executing the command `SHOW HOSTS (GRAPH | STORAGE | META)` with the service name.
```

Comment:
```suggestion
    - For the Nebula Graph cluster installed with the source code, the version of the cluster will not be displayed in the output after executing the command `SHOW HOSTS (GRAPH | STORAGE | META)` with the service name.
```

Reply:
will keep `a`, because it is not a specified cluster，it's generic.